README.md

```diff
@@ -83,3 +84,16 @@ IBM_DB2_EXPORTER_DB="database"
 
 ./ibm_db2_exporter
 ```
+
+# Troubleshooting
+
+If you get this error message:
+
+`ping is failing: verify DSN is correct and DB2 is running properly`
+
+It means the exporter is unable to connect to DB2, however it doesn't know why. To fix this error, please ensure the following:
+
+- Verify that the DSN being used by the exporter is correct for the instance/database of DB2 being monitored.
+- Verify that DB2 is running and all of it's communication protocols are activated.
+
+After making any necessary changes restart the exporter.
```

Comment:
Do you think it'd be appropriate to include the netstat command to help them troubleshoot?